project/Dependencies.scala

```diff
@@ -0,0 +1,103 @@
+    "com.typesafe.slick"           %% "slick-hikaricp"                % slickVersion exclude ("org.slf4j", "slf4j-api"),
+    "com.typesafe.slick"           %% "slick-codegen"                 % slickVersion,
+    "com.github.tminglei"          %% "slick-pg"                      % "0.18.0",
+    "com.kubukoz"                  %% "slick-effect"                  % "0.1.0" exclude ("com.typesafe.slick", "slick"),
+    "org.postgresql"               % "postgresql"                     % "42.1.4",
+    "com.github.scopt"             %% "scopt"                         % "4.0.0-RC2",
+    "io.scalaland"                 %% "chimney"                       % "0.3.1",
+    "com.madgag.spongycastle"      % "core"                           % "1.58.0.0",
+    "org.scorexfoundation"         %% "scrypto"                       % "2.1.7",
+    "org.scorexfoundation"         %% "scorex-util"                   % "0.1.6",
+    "com.muquit.libsodiumjna"      % "libsodium-jna"                  % "1.0.4" exclude ("org.slf4j", "slf4j-log4j12") exclude ("org.slf4j", "slf4j-api"),
+    "net.java.dev.jna"             % "jna"                            % "5.5.0", //see https://github.com/muquit/libsodium-jna/#update-your-projects-pomxml
+    "com.github.alanverbner"       %% "bip39"                         % "0.1",
+    "com.rklaehn"                  %% "radixtree"                     % "0.5.1",
+    "fr.acinq"                     %% "bitcoin-lib"                   % "0.9.18-SNAPSHOT",
+    "com.nrinaudo"                 %% "kantan.csv-generic"            % kantanCsvVersion,
+    "com.nrinaudo"                 %% "kantan.csv-java8"              % kantanCsvVersion,
+    "com.typesafe.akka"            %% "akka-testkit"                  % akkaVersion % Test exclude ("com.typesafe", "config"),
+    "com.typesafe.akka"            %% "akka-http-testkit"             % akkaHttpVersion % Test exclude ("com.typesafe", "config"),
+    "org.scalatest"                %% "scalatest"                     % "3.0.5" % "it, test",
+    "com.stephenn"                 %% "scalatest-json-jsonassert"     % "0.0.3" % "it, test",
+    "org.scalamock"                %% "scalamock"                     % "4.1.0" % "it, test",
+    "org.testcontainers"           % "postgresql"                     % "1.12.3" % "it, test",
+    "com.softwaremill.diffx"       %% "diffx-scalatest"               % "0.3.3" % "it, test",
+    compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
+    "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
+  )
+
+  val conseilCommonExclude = Seq(
+    "org.consensusresearch" %% "scrypto"
+  )
+
+  val conseilApiInclude = Seq(
+    "org.scalatest"     %% "scalatest"         % "3.0.5"         % "it, test",
```

Comment:
Isn't this somehow transitively inherited via module dependencies?

And the same for a few entries here-after...